icons/saxobank.svg

```diff
@@ -0,0 +1 @@
//www.w3.org/2000/svg"><title>Saxo Bank</title><path d="M2.508 6.791c-1.342.004-2.22.815-2.22 1.793 0 1.989 3.21 1.258 3.21 2.396 0 .4-.438.668-1.074.668-.67 0-1.104-.366-1.29-.93L0 11.083c.265.967 1.19 1.592 2.393 1.592 1.338 0 2.316-.748 2.316-1.766 0-2.061-3.232-1.39-3.232-2.426 0-.421.41-.67.959-.67.6 0 .912.288 1.076.612l1.119-.36c-.298-.796-1.12-1.273-2.123-1.273Zm18.508.004c-1.677 0-2.979 1.323-2.979 2.941 0 1.618 1.302 2.942 2.979 2.942 1.676 0 2.984-1.324 2.984-2.942s-1.308-2.941-2.984-2.941Zm-5.258.088L14.572 8.65l-1.185-1.763h-1.352l1.861 2.736-2.017 2.965h1.318l1.358-2.022 1.355 2.022h1.358L15.23 9.596l1.844-2.713Zm-7.68.004-2.428 5.7h1.243l.496-1.196h2.388l.49 1.19.782-1.153L9.11 6.887zm12.938.98c.996 0 1.752.782 1.752 1.87 0 1.086-.756 1.868-1.752 1.868-.997 0-1.75-.782-1.75-1.869s.768-1.869 1.75-1.869zm-12.44.608.785 1.927H7.795Zm6.291 6.843a.87.87 0 0 0-.299.069.571.571 0 0 0-.24.19.467.467 0 0 0-.086.282c0 .131.044.237.131.317.087.08.203.138.356.178l.207.054c.065.018.12.037.171.059a.31.31 0 0 1 .116.08.174.174 0 0 1 .045.119.208.208 0 0 1-.05.139.28.28 0 0 1-.126.091.518.518 0 0 1-.19.032.507.507 0 0 1-.187-.032.327.327 0 0 1-.135-.095.284.284 0 0 1-.059-.16h-.328a.57.57 0 0 0 .096.31c.062.084.14.148.246.192a.953.953 0 0 0 .371.066.958.958 0 0 0 .375-.07.534.534 0 0 0 .237-.188.506.506 0 0 0 .084-.285.458.458 0 0 0-.043-.207.45.45 0 0 0-.122-.152.693.693 0 0 0-.173-.106.997.997 0 0 0-.202-.064l-.17-.045a.584.584 0 0 1-.113-.033.434.434 0 0 1-.101-.047.228.228 0 0 1-.073-.073.163.163 0 0 1-.025-.097c0-.047.014-.092.04-.125a.277.277 0 0 1 .112-.082.443.443 0 0 1 .178-.033c.098 0 .179.02.24.064a.247.247 0 0 1 .106.184h.32a.512.512 0 0 0-.088-.278.627.627 0 0 0-.232-.185.787.787 0 0 0-.342-.069h-.047zm-10.123.026v1.836h1.2v-.28h-.868v-.5h.799v-.279h-.799v-.498h.862v-.28zm2.692 0v1.832h.33v-1.832zm.988 0v1.836h.332v-1.252h.017l.862 1.252h.287v-1.836h-.32v1.25h-.018l-.861-1.25zm1.924 0 .646 1.836h.412l.643-1.836h-.363l-.477 1.443h-.018l-.48-1.443zm2.164 0v1.836h1.199v-.28h-.865v-.5h.797v-.279h-.797v-.498h.857v-.28zm3.398 0v.279h.569v1.557h.326v-1.557h.568v-.28zm1.92 0v1.836h1.2v-.28h-.866v-.5h.797v-.279h-.797v-.498h.86v-.28Zm-14.955.004v1.832h.742a.833.833 0 0 0 .35-.067.476.476 0 0 0 .207-.177.467.467 0 0 0 .068-.254.45.45 0 0 0-.058-.237.41.41 0 0 0-.338-.21v-.018a.612.612 0 0 0 .164-.074.356.356 0 0 0 .117-.127.4.4 0 0 0 .043-.194.437.437 0 0 0-.262-.414.787.787 0 0 0-.33-.06zm16.762 0v1.832h.623a.972.972 0 0 0 .476-.11.752.752 0 0 0 .305-.316c.07-.135.105-.301.105-.49 0-.19-.036-.354-.105-.493a.728.728 0 0 0-.303-.316.95.95 0 0 0-.468-.107zm-16.43.271h.334c.098 0 .172.023.223.07.05.044.076.105.076.174a.244.244 0 0 1-.15.229.365.365 0 0 1-.157.033h-.326Zm16.762.02h.283a.63.63 0 0 1 .312.068.423.423 0 0 1 .19.207.817.817 0 0 1 .064.35c0 .138-.02.253-.064.347a.456.456 0 0 1-.193.211.662.662 0 0 1-.32.07h-.272Zm-16.762.726h.363c.07 0 .127.01.178.035a.275.275 0 0 1 .156.252c0 .073-.025.13-.08.178-.054.047-.142.07-.261.07h-.356Z"/></svg>
```

Comment:
Icons is matching perfectly now, thank you! There is a lot of extraneous nodes though in the "BE INVESTED" part. Mostly, the `B`, `S` and the `D`. It is not always easy to remove those kind of extra nodes. Can you give this a try and let me know?

Reply:
@service-paradis Will need help on this one

Reply:
Yeah, it is not easy. I just gave this a try and ends up with the following:
```svg
<svg xmlns="http://www.w3.org/2000/svg" xml:space="preserve" viewBox="0 0 24 24"><path d="M2.508 6.791c-1.342.004-2.22.815-2.22 1.793 0 1.989 3.21 1.258 3.21 2.396 0 .4-.438.668-1.074.668-.67 0-1.104-.366-1.29-.93L0 11.083c.265.967 1.19 1.592 2.393 1.592 1.338 0 2.316-.748 2.316-1.766 0-2.061-3.232-1.39-3.232-2.426 0-.421.41-.67.959-.67.6 0 .912.288 1.076.612l1.119-.36c-.298-.796-1.12-1.273-2.123-1.273zm18.508.004c-1.677 0-2.979 1.323-2.979 2.941 0 1.618 1.302 2.942 2.979 2.942 1.676 0 2.984-1.324 2.984-2.942s-1.308-2.941-2.984-2.941zm-5.258.088L14.572 8.65l-1.185-1.763h-1.352l1.861 2.736-2.017 2.965h1.318l1.358-2.022 1.355 2.022h1.358L15.23 9.596l1.844-2.713zm-7.68.004-2.428 5.7h1.243l.496-1.196h2.388l.49 1.19.782-1.153L9.11 6.887Zm12.938.98c.996 0 1.752.782 1.752 1.87 0 1.086-.756 1.868-1.752 1.868-.997 0-1.75-.782-1.75-1.869s.768-1.869 1.75-1.869zm-12.44.608.785 1.927H7.795Zm6.291 6.843c-.307.003-.64.226-.625.541 0 .131.044.237.131.317.122.156.581.224.734.291.11.04.164.114.161.199 0 .211-.244.273-.366.262-.16.01-.378-.08-.381-.287h-.328c.02.45.423.57.713.568.349.005.7-.165.696-.543.005-.337-.323-.488-.54-.529l-.17-.045c-.135-.017-.33-.107-.312-.25-.008-.169.191-.242.33-.24.098 0 .179.02.24.064.061.042.1.11.106.184h.32c-.017-.41-.407-.543-.662-.532zm-10.123.026v1.836h1.2v-.28h-.868v-.5h.799v-.279h-.799v-.498h.862v-.28zm2.692 0v1.832h.33v-1.832zm.988 0v1.836h.332v-1.252h.017l.862 1.252h.287v-1.836h-.32v1.25h-.018l-.861-1.25zm1.924 0 .646 1.836h.412l.643-1.836h-.363l-.477 1.443h-.018l-.48-1.443zm2.164 0v1.836h1.199v-.28h-.865v-.5h.797v-.279h-.797v-.498h.857v-.28zm3.398 0v.279h.569v1.557h.326v-1.557h.568v-.28zm1.92 0v1.836h1.2v-.28h-.866v-.5h.797v-.279h-.797v-.498h.86v-.28zm-14.955.004v1.832h.742c.504-.011.619-.283.625-.498a.426.426 0 0 0-.396-.447v-.018c.084-.003.335-.136.324-.395.015-.194-.129-.48-.592-.474zm16.762 0v1.832h.623c.521.004.894-.334.886-.916 0-.6-.36-.918-.876-.916zm-16.43.271h.334c.226-.004.296.12.299.244.001.137-.107.27-.307.262h-.326Zm16.762.02h.283c.523.002.565.402.566.625-.005.242-.06.63-.577.628h-.272zm-16.762.726h.363c.265 0 .338.195.334.287.002.103-.06.253-.341.248h-.356Z"/></svg>
```